discrete_math/euclidean_algorithm/euclidian.cpp

```diff
@@ -0,0 +1,46 @@
+#include <iostream>
+#include <format>
+#include <print>
+#include <numeric>
+
+//int gcd(int a, int b) {
+//    while (b != 0) {
+//        int temp = b;
+//        std::cout << "(" << a << ", " << b << ") = \n";
+//        b = a % b;
+//        std::cout << "Reszta z dzielenia: " << b << "\n";
+//        a = temp;
+//    }
+//    return a;
+//}
+
+void tablica() {
+    int n;
+    std::cout << "Podaj dla ilu liczb chcesz policzyc NWD: ";
+    std::cin >> n;
```

Comment:
as we talked: read data from command line that is for example <name_of_your_program> 2 4 5 should run extended Euclidean algo for 2 4 5